x/oracle/keeper/prices.go

```diff
@@ -99,6 +106,15 @@ func (k Keeper) GetMultipleAssetsPrices(ctx sdk.Context, assets map[string]inter
 			}
 		} else {
 			v, _ := sdkmath.NewIntFromString(price.Price)
+			// for tokens really have 0 price, it should be removed from assets support, not here to provide zero price
+			if v.IsNil() || v.LTE(sdkmath.ZeroInt()) {
+				info = info + assetID + " "
+				prices[assetID] = types.Price{
+					Value:   sdkmath.NewInt(types.DefaultPriceValue),
+					Decimal: types.DefaultPriceDecimal,
```

Comment:
I found that the `DefaultPriceDecimal` is 0, which means the default price would be 1 USD. This could incentivize a DDoS attack on the oracle to increase the value of assets priced below 1 USD. 
Should we set it to 6 to reduce this attack motivation. But we need to consider whether the initial voting power is enough when starting the local network through `local_node.sh` if we make this change.